Sources/NIOInstrumentation/BaggageContextInboundHTTPHandler.swift

```diff
@@ -0,0 +1,28 @@
+import BaggageContext
+import Instrumentation
+import NIO
+import NIOHTTP1
+
+public final class BaggageContextInboundHTTPHandler: ChannelInboundHandler {
+    public typealias InboundIn = HTTPServerRequestPart
+    public typealias InboundOut = HTTPServerRequestPart
+
+    private let instrument: Instrument<HTTPHeaders, HTTPHeaders>
+    private var onBaggage: (BaggageContext) -> Void
```

Comment:
onBaggage sounds a bit weird, we're missing a verb here; it's more like onBaggageExtracted?

Reply:
Yes, I like that suggestion.